core/domain/entities/editor/BlockInterface.php

```diff
@@ -70,18 +81,6 @@ public function appliesToPostType($post_type);
     public function getEditorContainer();
```

Comment:
Offhand I'm not sure what the purpose of this method is for (its not documented), but also other elements in this interface have been genericized, would that apply to this as well?

Reply:
This was based on code that you had added in the initial Gutenberg integration add-on. 
It wasn't documented there and I couldn't quite grok what it did, but assumed it was necessary.
I can remove until we determine if it is even needed still.